docs/seed-jobs.md

```diff
@@ -3,10 +3,10 @@
 Requires `job-dsl` >= 1.74
 
 Configuration is not just about setting up Jenkins master, it's also about creating an initial set of jobs.
-For this purpose, we delegate to the popular [job-dsl-plugin](https://wiki.jenkins.io/display/JENKINS/Job+DSL+Plugin)
+For this purpose, we delegate to the popular [job-dsl-plugin](https://plugins.jenkins.io/job-dsl)
 and run a job-dsl script to create an initial set of jobs.
 
-Be aware that you also need [Configuration as Code Support](https://plugins.jenkins.io/configuration-as-code-support) besides [Configuration as Code](https://plugins.jenkins.io/configuration-as-code) plugin already installed, otherwise the `jobs` root element cannot be parsed.
+Be aware that you need [job-dsl-plugin](https://plugins.jenkins.io/job-dsl) version 1.74 or newer besides [Configuration as Code](https://plugins.jenkins.io/configuration-as-code) plugin already installed, otherwise the `jobs` root element cannot be parsed.
```

Comment:
Remove this whole section? it's already specified at the top? (added after this section was written)
```suggestion
```

Reply:
👍